### PR TITLE
Adds seashells to the game.

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

See title.

Also adds some seashells to the beach away mission and the beach dome.

## Why It's Good For The Game

It's a harmless decorative item. People can use it in their mapping and whatnot.

## Changelog
:cl:
add: Seashells have been added to the game.
/:cl:

### CODE REVIEW

**Comment by partykp** [comment]
Please add 3 seashells to the bathrooms just for the reference

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -1407,3 +1407,18 @@
 
 /obj/item/toy/dummy/GetVoice()
 	return doll_name
+
+/obj/item/toy/seashell
+	name = "seashell"
+	desc = "May you always have a shell in your pocket and sand in your shoes. Whatever that's supposed to mean."
+	icon = 'icons/misc/beach.dmi'
+	icon_state = "shell1"
```
Comment (ATH1909) [inline code/game/objects/items/toys.dm]: maybe we should give it a custom grind_results list?

**Comment by actioninja** [comment]
go away bikeshedders

**Comment by optimumtact** [comment]
sick power creep, you are gitbanned

File: code/game/objects/items/toys.dm [hunk 2]
```diff
@@ -1407,3 +1407,18 @@
...
+	var/static/list/possible_colors = list("" =  2, COLOR_PURPLE_GRAY = 1, COLOR_OLIVE = 1, COLOR_PALE_BLUE_GRAY = 1, COLOR_RED_GRAY = 1)
+
+/obj/item/toy/seashell/Initialize()
+	. = ..()
+	pixel_x = rand(-5, 5)
+	pixel_y = rand(-5, 5)
+	icon_state = "shell[rand(1,3)]"
+	color = pick(possible_colors)
```
Comment (SpaceManiac) [inline code/game/objects/items/toys.dm]: did you mean for this to be a `pickweight`, given that `possible_colors` is an associative list?

**Review by SpaceManiac — CHANGES_REQUESTED** [review]

**Review by SpaceManiac — APPROVED** [review]

**Pull request merged**